pkg/argsbuilder/serving_tensorflow.go

```diff
@@ -94,6 +94,7 @@ func (s *TensorflowServingArgsBuilder) AddCommandFlags(command *cobra.Command) {
 	command.Flags().StringVar(&s.args.ModelPath, "model-path", "", "the model path for serving in the container")
 
 	command.Flags().StringVar(&s.args.ModelConfigFile, "modelConfigFile", "", "Corresponding with --model_config_file in tensorflow serving")
```

Comment:
please use model-config-file to replace modelConfigFile

Reply:
done